src/allmydata/crypto/rsa.py

```diff
@@ -112,14 +123,14 @@ def der_string_from_signing_key(private_key):
     :returns: bytes representing `private_key`
     """
     _validate_private_key(private_key)
-    return private_key.private_bytes(
+    return private_key.private_bytes( # type: ignore[attr-defined]
```

Comment:
Try as I might, I could not convince mypy that the `private_bytes` method exists.